<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar{
        border: none;  
        text-align: center;
        background: rgba(0, 0, 0,0); 
        border-radius:8px;
} 

QProgressBar::chunk {
        background: rgb(244, 88, 56); 
        border-radius:6px;
} 
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>221</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 16pt &quot;幼圆&quot;;

border-image: url(:/brower_003.png);
border-color: rgba(255, 255, 255, 0);
border:0;</string>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/brower_002.png);</string>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>91</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0;
font: 14pt &quot;华文琥珀&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
